docs/src/pages/index.tsx

```diff
@@ -2,7 +2,7 @@ import type { NextPage } from "next";
 
 import HeroBlock from "../components/landingBlocks/HeroBlock";
 import FeatureCard from "../components/ui/FeatureCard";
-import GetStartedHardhat from "../components/landingBlocks/CTABlock";
+import GetStartedHardhat from "../components/landingBlocks/GetStartedBlock";
```

Comment:
Hm... the new name of a file is really better but also let's make the imported component to have the same name